### DISCUSSION TITLE
Countdown with background music option.

### DISCUSSION THREAD

**Discussion by Chipperchap** [AntonyCorbett/OnlyT#443] [Ideas]
Feature request. 

A selection of music files (sjjm) are preloaded and can be selected to play as background music or randomly played. The music could be sjj vocal renditions in a variety of languages for example. 
In an ideal world the music snaps to the end so that the fade out matches the close of the countdown. This would mean that a 5:15 minute music file would have the :15 seconds removed from the start for a five minute countdown (or any other countdown time selection) so that it finishes as the countdown finishes.

Alternatively the background music just fades at its current state so a music file that is 5:15 will fade out with 15 seconds to go with a five minute countdown; fade out with 2:15 to go for a three minute countdown for example. 

We are the only congregation around my location that uses OnlyT countdown timer and background music to begin the meeting so probably not a common request. This feature would be useful for our current ZOOM meetings and when we eventually return to the Kingdom Hall.

**Comment by mvpapen** [comment]
I am looking for this feature for over a year, please include individual sound volume control.
right now if you play background music over Jw app, and you test any other video or even picture you interrupt the Music. Also you have to turn down the Sound either in the app or amp, and it never fails  that when starting the 1 meeting song that the sound level is off. We would greatly appreciated this function.
